java/src/client/turntracker/TurnTrackerController.java

```diff
@@ -68,13 +70,22 @@ public void endTurn() {
      * Gets the correct color of the players whos turn it is, also inits all the players
      */
     private void initFromModel() {
-        getView().setLocalPlayerColor(GameManager.getGame().getPlayerInfo().getColor());
-        List<Player> players = GameManager.getGame().getClientModel().getPlayers();
+        CatanColor color;
+        List<Player> players = null;
+        try{
+            color = GameManager.getGame().getPlayerInfo().getColor();
+            getView().setLocalPlayerColor(color);
+            players = GameManager.getGame().getClientModel().getPlayers();
+
+            for (Player p : players) {
+                if (p == null) continue;
```

Comment:
It's a serious problem if there is a `null` player. We should fail quickly when that happens.